src/app/pages/test/TestContainer.js

```diff
@@ -1,51 +1,12 @@
-import PropTypes from 'prop-types'
-import { autobind } from 'core-decorators'
-import { Component } from 'react'
-import { compose } from 'redux'
-import { reduxForm } from 'redux-form'
-
-// import { connectResource, connectFormResource } from 'common/utils/resource'
+import { finalForm } from '@ds-frontend/resource'
 import Test from './Test'
 
 
-const propTypes = {
-  handleSubmit: PropTypes.func.isRequired,
-}
-
-class TestContainer extends Component {
-  @autobind
-  onSubmit(data) {
-    console.log('submitted', data) // eslint-disable-line no-console
-  }
-
-  render() {
-    const { handleSubmit, ...props } = this.props
-    return <Test
-      {...props}
-      onSubmit={handleSubmit(this.onSubmit)}
-    />
-  }
-}
-
-TestContainer.propTypes = propTypes
-
-/*
-let testResource = {
-  namespace: 'leads',
-  endpoint: 'leads/e4831306-9e2e-41b7-bb77-514318aa51ba',
-}
-*/
-
-export default compose(
-  // connectResource(testResource),
-  // connectFormResource(testResource),
-
-  reduxForm({
-    form: 'test',
-    initialValues: {
-      firstName: 'Stephen',
-      lastName: 'Hawking',
-      salutation: 'mr',
-    },
-  }),
-)(TestContainer)
+export default finalForm({
+  initialValues: {
+    firstName: 'Stephen',
+    lastName: 'Hawking',
+    salutation: 'mr',
+  },
+  onSubmit: console.log,
+})(Test)
```

Comment:
1. please move resource to separate file. 
2. this file is not a container. rename it to `Test.js`